bionic/decorators.py

```diff
@@ -440,6 +441,31 @@ def returns(out_descriptor):
     )
 
 
+def aip_resource(resource):
```

Comment:
Let's add the same disclaimer as we have on `accepts` and `returns`:

    This decorator is currently experimental and does not have any additional
    user-facing documentation. It may change in non-backwards-compatible ways.